lib/redis-bus.js

```diff
@@ -0,0 +1,59 @@
+const Redis = require('ioredis')
+
+/**
+ * This class extends the EventEmitter to act as a pub/sub message bus,
+ * allowing payloads to be received by one instance of Smee and sent by
+ * others. This allows Smee to be multi-instanced!
+ */
+module.exports = class RedisBus {
+  /**
+   * @param {import('events').EventEmitter} events
+   */
+  constructor (events) {
+    this.events = events
+
+    // If Redis isn't enabled, don't try to connect
+    if (process.env.REDIS_URL) {
```

Comment:
Just a tiny nitpick, maybe a personal preference, but I prefer to return early instead of wrapping bigger chunks of code in an if statement

So I would do

```js
if (!process.env.REDIS_URL) return
```

and then continue without antother indentation

Reply:
ooh good idea 👍